networks/rpc/client.go

```diff
@@ -458,6 +458,20 @@ func (c *Client) newMessage(method string, paramsIn ...interface{}) (*jsonrpcMes
 	return &jsonrpcMessage{Version: "2.0", ID: c.nextID(), Method: method, Params: params}, nil
 }
 
+func (c *Client) getMessageSize(method string, args ...interface{}) (int, error) {
```

Comment:
Why don't you put this function in `networks/rpc/websocket_test.go"?

Reply:
I put in `client.go` because it could be used at some place less. A new RPC call or a new test code in `tests` package could be expected.